developers/contributing/contributing.md

```diff
@@ -166,6 +160,12 @@ then you just add a line to every git commit message:
 
 using your real name (sorry, no pseudonyms or anonymous contributions.)
 
+If your commit contains code from others as well, please ensure that they certify the DCO as well and add them with an "Also-By" line to your commit message:
+
+    Also-by: Ted Nerd <ted.nerd@email.com> (github: github_handle_ted)
+    Also-by: Sue Walker <sue.walker@email.com> (github: github_handle_sue)
+    Signed-off-by: Joe Smith <joe.smith@email.com> (github: github_handle_joe)
```

Comment:
Yes, at least that's also how the Eclipse Foundation handles it.
The Signed-off-by should always be the last line (like a signature on a document).